admin/tools/docker/configured/Dockerfile.tpl

```diff
@@ -0,0 +1,15 @@
+FROM {{DOCKER_IMAGE_OPT}}
+MAINTAINER Fabrice Jammes <fabrice.jammes@in2p3.fr>
+
+WORKDIR /qserv
+
+USER qserv
+
+RUN bash -c ". /qserv/stack/loadLSST.bash && setup qserv -t qserv-dev && /qserv/scripts/configure.sh {{NODE_TYPE_OPT}} {{MASTER_FQDN_OPT}}"
```

Comment:
Good idea :+1: (consul might also work if we have more configuration parameters), but I would prefer doing that in a ticket dedicated to configuration in order to explore all possible solutions...